packages/itinerary-body/src/AccessLegBody/index.tsx

```diff
@@ -137,7 +140,13 @@ class AccessLegBody extends Component<Props, State> {
                     aria-expanded={expanded}
                     onClick={this.onStepsHeaderClick}
                   >
-                    <Duration seconds={leg.duration} />
+                    <Duration
```

Comment:
As is, an undesired line break is introduced in the button somehow (see screenshot), can you fix that please?

<img width="113" alt="image" src="https://github.com/opentripplanner/otp-ui/assets/56846598/46edbecc-6eda-41db-b1ea-c0b323d0e408">